examples/scan.js

```diff
@@ -0,0 +1,34 @@
+var redis  = require("redis"),
+    client = redis.createClient();
+
+var cursor = 0;
+
+function scan() {
+    client.scan(
+        cursor,
+        "MATCH", "q:job:*",
+        "COUNT", "10",
+        function(err, res) {
+            if (err) throw err;
+
+            // Update the cursor position for the next scan
+            cursor = res[0];
+
+            // From <http://redis.io/commands/scan>:
+            // An iteration starts when the cursor is set to 0,
+            // and terminates when the cursor returned by the server is 0.
+            if (cursor === 0) {
+                return console.log('Iteration complete');
+            } else {
+                // Remember, more keys than COUNT or no keys may be returned
```

Comment:
Also, the docs don't say that 'more than COUNT or 0' will be returned. It just says that COUNT should be roughly the number of items returned. Could be more. Could be less.